meeting-notes/2019/Q1/2019-01-02--gui-and-in-web-browsers-weekly.md

```diff
@@ -0,0 +1,115 @@
+
+    -   _@terichadbourne_
+    -   _@lidel_
+    -   _@jimpick_
+    -   _@olizilla_
+    -   _@hacdias_
+
+  
+
+
+**Moderator checklist**
+
+-   Ensure that there is a notetaker
+-   Start recording
+-   Call for additional agenda items
+
+  
+
+
+**Agenda**
+
+_General discussions, decisions, etc._
+
+-   Start with Team Updates and Demos
+
+  
+
+
+**Team Updates**
+
+@lidel
+
+-   Done:
+    -   Holidays :-)
+    -   Discussions around accessing API of remote IPFS node - [in-web-browsers/issues/137](https://github.com/ipfs/in-web-browsers/issues/137) 
+        -   @Gozala shared some relevant ideas in  
+            [Progressive peer-to-peer web applications (PPWA)](https://via.hypothes.is/https://gozala.hashbase.io/posts/Native%20talk.html)  and tested initial assumptions in [in-web-browsers/issues/137#comment-450530548](https://github.com/ipfs/in-web-browsers/issues/137#issuecomment-450530548) 
+    -   Switched Companion to modern Clipboard API, PDF and SVG work again - [ipfs-companion/pull/649](https://github.com/ipfs-shipyard/ipfs-companion/pull/649)
+    -   Added a warning about mixed-content when gateway URL is not a Secure Context - [ipfs-companion/pull/650](https://github.com/ipfs-shipyard/ipfs-companion/pull/650) 
+    -   Started discussion about the future of DNSLink cache - [ipfs-companion/pull/646](https://github.com/ipfs-shipyard/ipfs-companion/pull/646#issuecomment-450879014)  + [go-ipfs/issues/5884](https://github.com/ipfs/go-ipfs/issues/5884) 
+-   Next:
+    -   Finalize OKRs
+    -   tackle holiday backlog
+    -   release Companion
+
+  
+
+
+@hacdias
+
+-   Done:
+    -   Fixed issue where we were calling methods that only exist on macOS: <https://github.com/ipfs-shipyard/ipfs-desktop/pull/761>
+    -   Fixed the links not opening in external windows: <https://github.com/ipfs-shipyard/ipfs-desktop/pull/769>
+    -   Updated to Electron 4: <https://github.com/ipfs-shipyard/ipfs-desktop/pull/769>
+    -   Fixed the black screenshot issue: <https://github.com/ipfs-shipyard/ipfs-desktop/pull/769>
+    -   Improved the screenshots experience: there will be now a notification telling the user a screenshot was taken and if they click on it, Web UI will open on that screenshots. If the user has a multi-monitor setup, the screenshots will be in a directory.
+        -   <https://github.com/ipfs-shipyard/ipfs-desktop/pull/759>
+        -   <https://github.com/ipfs-shipyard/ipfs-desktop/pull/775>
+-   Blocked:
+    -   Waiting for macOS guys on this PR: <https://github.com/ipfs-shipyard/ipfs-desktop/pull/770>
+-   Next:
+    -   Go through Desktop’s issue list and see what can be closed and fixed.
+    -   Prioritize native OS integrations: <https://github.com/ipfs-shipyard/ipfs-desktop/issues/679>
+    -   Work on that ^^
+    -   🧙‍♂️
+
+@alanshaw
+-   Done:
+    -   JS IPFS 0.34 release dance 💃
+    -   Holidays
+-   Blocked:
+    -   JS IPFS 0.34 release blocked on stack overflow issues with latest libp2p
```

Comment:
Are these issues documented? Could you link them here please?

Reply:
Will DHT be part of this release again?

Reply:
Just one issue: https://github.com/libp2p/js-libp2p-switch/issues/287

PRs resolving it:
* [x] https://github.com/libp2p/js-libp2p-switch/pull/297
* [x] https://github.com/libp2p/js-libp2p-crypto/pull/136
* [ ] https://github.com/ipfs/js-ipfs-bitswap/pull/184

This was discovered only when getting interop tests in good shape for the 0.34 release: https://github.com/ipfs/interop/pull/51

As of right now, DHT is still not part of this release. Once we get the interop tests back in good shape we can evaluate how much work is remaining on the DHT endeavour.